tsl/src/compression/compression.c

```diff
@@ -814,6 +817,7 @@ index_scan_sequence_number(Relation table_rel, Oid index_oid, ScanKeyData *scank
 	index_scan->xs_want_itup = true;
 	index_rescan(index_scan, scankey, num_scankeys, NULL, 0);
 
+	/* TODO: add sanity check */
```

Comment:
Can you expand on this?

Reply:
that isnt even code added by this PR, removed